main.tf

```diff
@@ -114,17 +114,6 @@ data "aws_security_group" "ssh_rdp" {
   }
 }
 
-data "aws_security_group" "rds" {
```

Comment:
Was this not being used for something?

Reply:
This is per the decision the AWS SWAT team made. Now that they've removed it from some account, the ACS module doesn't work in those accounts (like the payment systems account).